TERMS.md

```diff
@@ -111,6 +111,10 @@ Do not use. Use *deny list* instead.
 
 Avoid using. Use *service outage* or *blocked* instead.
 
+**BM25**
+
+A ranking function used to estimate the relevance of documents to a given search query.
```

Comment:
We may want to reference here how BM25 is an extension of TF-IDF:

`A ranking function used to estimate the relevance of documents to a given search query. BM25 extends TF-IDF by normalizing document length.`

Open to thoughts but it may help close the gaps for some people.